yt_dlp/extractor/zaiko.py

```diff
@@ -0,0 +1,67 @@
+from .common import InfoExtractor
+from ..utils import (
+    extract_attributes,
+    int_or_none,
+    str_or_none,
+    traverse_obj,
+    unescapeHTML,
+    url_or_none,
+)
+
+
+class ZaikoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?zaiko\.io/event/(?P<id>\d+)/stream(?:/\d+)+'
```

Comment:
apparently there are artist subdomains too, e.g. https://vark-live.zaiko.io/
```suggestion
    _VALID_URL = r'https?://(?:[\w-]+\.)?zaiko\.io/event/(?P<id>\d+)/stream(?:/\d+)+'
```